### ISSUE TITLE
[TC6, Foci] Easier access to crafting interface

### ISSUE THREAD

**Issue by AnrDaemon** [Azanor/thaumcraft-suggestions#223]
1. Make crafting grid zoomable.
2. Put player's inventory to the bottom, use free space at the left to show options in 3 columns: 
    - Effects
    - Directions
    - Modifiers
3. Don't reset the scroll position of the effects panel with each pick.

**Comment by Daderic** [comment]
I agree, and I also think that the table should be able to access an inventory next to it. That way, I can just put my crystals in there and be done with it.
